tgui/packages/tgui/interfaces/Vending.tsx

```diff
@@ -42,7 +43,7 @@ type UserData = {
   name: string;
   cash: number;
   job: string;
-  department: string;
+  department_bitflag: string;
```

Comment:
Fixing some improper conversions I made in the previous pr. Reverting to actually get the correct variables per #7559